husky_directory/models/search.py

```diff
@@ -29,19 +28,24 @@ class SearchDirectoryInput(DirectoryBaseModel):
     netid: Optional[str] = Field(None)
 
 
+class PhoneContactMethods(DirectoryBaseModel):
+    # These aliases are for humans, instead of for computers, so use
+    # human conventions for the front end to display.
+    phones: List[str] = Field([], alias="phone")
+    faxes: List[str] = Field([], alias="fax")
+    voice_mails: List[str] = Field([], alias="voicemail")
+    touch_dials: List[str] = Field([], alias="tdd")
+    mobiles: List[str] = Field([], alias="mobile")
+    pagers: List[str] = Field([], alias="pager")
+
+
 class Person(DirectoryBaseModel):
     name: str
-    phone: Optional[str]
+    phone_contacts: PhoneContactMethods = PhoneContactMethods()
```

Comment:
line 44
Great, only one phone field now. `phone` was type `Optional` before and with the new type `PhoneContactMethods `, without the `Optional` type is this field now required?

Reply:
No, a default is provided. If you look up `Optional` you'll find that it's just an alias that also accepts None.

Reply:
And remember: this is _output_, not input. You won't be building these objects.

Reply:
https://pydantic-docs.helpmanual.io/usage/types/#standard-library-types

Reply:
I highly recommend reading and familiarizing yourself with the pydantic docs. They're quite great. When I started learning pydantic I read them "cover to cover".